views/check-your-answers.html

```diff
@@ -59,12 +59,23 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application<
 
       <br>
       <form method="post">
-        {{ govukButton({
-          text: "Continue to payment",
-          attributes: {
-            "id": "submit"
-          }
-        }) }}
+        <div class="govuk-button-group">
+          {{ govukButton({
+            text: "Continue to payment",
+            attributes: {
+              "id": "submit"
+            }
+          }) }}
+
+          {{ govukButton({
```

Comment:
are you able to move the print button govukButton into the views/includes/print-button.html file?
and then have on line 70 {% include "includes/print-button.html" %}
might be worth a try as we can then have all the button code in the includes/print-button